backend/inspirehep/search/aggregations.py

```diff
@@ -252,6 +252,37 @@ def hep_rpp(order, title="Exclude RPP", agg_type="checkbox"):
     }
 
 
+def hep_self_author_claimed_papers_aggregation(
+    order, title="Claims", agg_type="checkbox",
+):
+    return {
+        "self_curated_relation": {
+            "filters": {
+                "filters": {
+                    "Claimed papers": {
+                        "nested": {
+                            "path": "authors",
+                            "query": {"term": {"authors.curated_relation": True}},
+                        }
+                    },
+                    "Unclaimed papers": {
+                        "nested": {
+                            "path": "authors",
+                            "query": {"term": {"authors.curated_relation": False}},
+                        }
+                    },
```

Comment:
not sure this solves anything, but why not use an `other` bucket for this part? https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-filters-aggregation.html#other-bucket

Reply:
also I don't see how you're restricting to the relevant author here. Doesn't this look for claims in all authors?